userManual/README.md

```diff
@@ -16,54 +16,115 @@ Explains:
+* Title of Event
 
-The date of the event is defaulted to the current day and the time is defaulted to the time you opened the page. 
+![Title of Event](designerSnapshots/title.PNG)
+* Description of Event
+
+![Description of Event](designerSnapshots/desc.PNG)
+* Photos (although this field does not have an asterisk it is a required field)
+
+![Description of Event](designerSnapshots/photos.PNG)
+
+##### Setting a Time and Date
+There is no default date, and if no date is selected then it is assumed that the Slide is a general announcement for the current day and the time is defaulted to the time you opened the page.
 
 ![](designerSnapshots/dateAndTime.PNG)
 
-To change the default date, select a different day on the calendar.
+To set a date date, select a day on the calendar.
 And to change the default time, click on the time and a clock will pop up:
 
 ![](designerSnapshots/changeTime.PNG)
 
-Select your new time and voila! To differentiate between AM and PM, click which option you want. 
+Select your new time and voila! To differentiate between AM and PM, click which option you want.
+
+##### Adding/Removing Photos on Slides
+Adding a photo to this application is much like any other application. 
+Click the plus button and find the photo you wish to show!
+Removing them is just as easy! Click the red 'X'.
+
+![Remove Photo](designerSnapshots/removePhoto.PNG)
+
+##### *I'm Finished Editing a Slide, Now What?*
+From this point on you can do a couple of things, click:
+* Clear, this option will clear your entire page
+
+![Clear](designerSnapshots/clear.PNG)
+* Preview, see what this does below
+
+![Preview](designerSnapshots/previow.PNG)
+* Submit, this option will bring you to the Preview page so you can see your slide. From here you will need to click submit again to make your Slide to be added to the Show View Page.
+
+![Submit](designerSnapshots/submit.PNG)
+
+###### Preview
+The Preview button will open up a preview so you can see what your Slide will look like when it is submitted.
+
+![Preview of Slide](designerSnapshots/preview.PNG)
+
+##### Customizing Different Fields
+Each field can be customized, just click the down arrow.
+
+![Down Arrow](designerSnapshots/downArrow.PNG)
 
+![Customize](designerSnapshots/customize.PNG)
+
+And once everything is set, it can look.... Cool?
+
+![Customized Design](designerSnapshots/customized.PNG)
+
+Okay, lets face it... I'm not good at designing things.
+
+##### Slide Settings
+There is also another thing that you may have noticed, it's the Slide Settings configuration. 
+Again, if you click the down arrow it will pop up with a new area.
+
+![Slide Settings](designerSnapshots/slideSettings.PNG)
+
+From here you can customize a few things:
+* Duration, the amount of 'air time' you wish for it to have on the Show View Page!
+The options range from 10seconds to 80seconds, in 10second intervals.
+
+![Duration](designerSnapshots/displaySeconds.PNG)
+* Wait, another Duration?
```

Comment:
_*QUEsTION*_, what is the purpose of this one?

Reply:
The second "Duration" is a typo. It specify that it changes the template

Reply:
But we don't have templates?